applications/conceptual-model-editor/src/app/diagram/reactflow/components/menu-options.tsx

```diff
@@ -43,40 +45,74 @@ export const useMenuOptions = () => {
         deleteHandler?: () => void;
         position?: string;
     }) => {
-        const { openDetailHandler, createProfileHandler, modifyHandler, removeFromViewHandler, deleteHandler } = props;
+        const { openDetailHandler, createProfileHandler, modifyHandler, removeFromViewHandler, deleteHandler, position } = props;
```

Comment:
At this point I would consider not using the spread and just use "props." in the code bellow.
We may need to have a guideline for this .. any though on when to spread and when not to?

Reply:
Some guide lines probably would be nice, there were already some places in the old code, where the value was used both from props and the spread variable. 
About specific guide-line. I don't know are there any "official" ones? I guess that one could consider length of names, the ratio of spread values to all available, number of usages, number of spread values, or maybe even types of spread values.

The safest choice would be of course to always use the props and don't use spread, but that's probably also not really ideal. Or just limit to some fixed number ... > 4 use props, else use spread. ...... just to name a some of the simple ones